viewer/src/views/threejs/cad/CadNode.ts

```diff
@@ -35,6 +39,7 @@ const updateVars = {
 export class CadNode extends THREE.Object3D {
   public readonly rootSector: SectorNode;
   public readonly modelTransformation: SectorModelTransformation;
+  public readonly _shading: Shading;
```

Comment:
Why is this field prefixed with _? Is it private?

Reply:
Ooops. Yes, let us keep this private for now.